spec/example_app/app/dashboards/blog/post_dashboard.rb

```diff
@@ -0,0 +1,33 @@
+require "administrate/base_dashboard"
+
+module Blog
+  class PostDashboard < Administrate::BaseDashboard
+    ATTRIBUTE_TYPES = {
```

Comment:
@nickcharlton  this is a generated dashboard by Administrate itself. It will be nice to fix it in the generator, not in generated files. I can do it in a another PR. What do you think?